src/plugins/rovpilot/public/js/rovpilot.js

```diff
@@ -89,8 +113,11 @@
         {
           axis: 
           {
-            update: function(value) {
-              rov.cockpit.emit('plugin.rovpilot.setThrottle', -1 * postProcessStickValues(value));
+            update: function(value, invert) {
+              var inversion = invert ? 1 : -1;
```

Comment:
This seems like a concern that should be buried in the input manager's handling of the events in its gamepad abstraction.  We should not have invert code throughout the plugins.  The "value" should already be inverted by the time the listener is called.